### PR TITLE
 update translate for class-and-style.md and forms.md.

### PR DESCRIPTION
**Pull request by tolerious**
There are some words didn't be translated in `class-and-style.html`  file.

### CODE REVIEW

File: src/v2/guide/class-and-style.md [hunk 1]
```diff
@@ -109,39 +109,39 @@ data: {
 <div v-bind:class="[{ active: isActive }, errorClass]">
 ```
 
-### With Components
+### 和组件一起使用
```
Comment (Jinjiang) [inline src/v2/guide/class-and-style.md]: 直接翻译成“用在组件上”怎么样？

File: src/v2/guide/class-and-style.md [hunk 2]
```diff
@@ -109,39 +109,39 @@ data: {
...
 
-> This section assumes knowledge of [Vue Components](components.html). Feel free to skip it and come back later.
+> 这个部分假设你已经有了对于[Vue组件](components.html)的一些了解。尽管跳过这段好了，过会儿再回来看也是可以的。
```
Comment (Jinjiang) [inline src/v2/guide/class-and-style.md]: 推荐译法：`这个章节假设你已经对 [Vue 组件](components.html)有一定的了解。当然你也可以跳过这里，稍后再回过头来看`

另外中英文之间要用空格隔开

Comment (tolerious) [inline src/v2/guide/class-and-style.md]: done.

File: src/v2/guide/class-and-style.md (hunk 1 above)
Comment (tolerious) [inline src/v2/guide/class-and-style.md]: sure, I think so.

**Pull request merged**